src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -23,6 +29,17 @@
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
 
+    // Temporary modules and modules list
+    private Module module1 = new Module(new ModuleCode("CS1231"), Optional.of(new ModuleName("Shezad")));
+    private Module module2 = new Module(new ModuleCode("CS2105"), Optional.of(new ModuleName("Networks")));
+    private Module module3 = new Module(new ModuleCode("Cs3333"), Optional.empty());
+
+
+    private final FilteredList<Module> filteredModules;
+    private final ObservableList<Module> internalList = FXCollections.observableArrayList(module1, module2, module3);
```

Comment:
i think no need for this `internalList` cause ure not using it anywhere. jus declare `observableArrayList` directly in the line below